java/src/jmri/jmrit/beantable/LogixTableAction.java

```diff
@@ -564,6 +659,199 @@ protected String helpTarget() {
     private boolean _newItem = false; // marks a new Action or Variable object was added
 
     /**
+     * Input selection names
+     */ 
+    public enum SelectionMode {
+        USEMULTI, USESINGLE, USECOMBO;
+    }
+    SelectionMode _selectionMode;
+    
+    // Retain the menu objects for the doClick() method.
+    JRadioButtonMenuItem menuMulti; 
+    JRadioButtonMenuItem menuSingle; 
+    JRadioButtonMenuItem menuCombo; 
+     
+    /**
```

Comment:
This is a general comment block, along with another one a bit later, that is not tied to a particular method.  Should these be formatted differently, such as just // lines, or maybe change  /** to /*?